packages/flame_markdown/example/pubspec.yaml

```diff
@@ -11,6 +11,7 @@ dependencies:
   flame_markdown: ^0.2.2+3
   flutter:
     sdk: flutter
+  markdown: ^7.1.1
```

Comment:
note: in order to provide custom documents, a user would need to add this as a direct dependency. another option is that we export all the relevant classes on flame_markdown; not sure what would be the best practice here

Reply:
If there aren't that many I think we should export them

Reply:
let me make a followup as a proposal and we can assess together - sadly I think there is quite a few